libs/blocks/commerce/commerce.js

```diff
@@ -0,0 +1,139 @@
+import { createTag, getConfig, loadScript } from '../../utils/utils.js';
+import { getTacocatEnv, runTacocat, buildCheckoutButton, getCheckoutContext, omitNullValues } from '../merch/merch.js';
```

Comment:
@Axelcureno : can you please add unit tests for this new `commerce` block? We want to make sure it's not breaking in case these imported functions are modified in the `merch` block. Not sure if it makes sense to externalize these functions into a separate `utils` script.

Reply:
Unit tests added ✅